controllers/internal/utils/annotations.go

```diff
@@ -0,0 +1,17 @@
+package utils
+
+// MergeAnnotations adds or updates annotations from a2 from a1 and returns merge
```

Comment:
Just a little one :)
```suggestion
// MergeAnnotations adds or updates annotations from source to target and returns merge result
```